compiler/circledump/src/OpPrinter.cpp

```diff
@@ -675,7 +690,6 @@ OpPrinterRegistry::OpPrinterRegistry()
   _op_map[circle::BuiltinOperator_SUB] = make_unique<SubPrinter>();
   _op_map[circle::BuiltinOperator_SUM] = make_unique<ReducerPrinter>();
   _op_map[circle::BuiltinOperator_TRANSPOSE_CONV] = make_unique<TransposeConvPrinter>();
-
```

Comment:
Seems this change is not related.

Reply:
It is:
https://github.com/Samsung/ONE/pull/1845#discussion_r434659541